x-pack/plugins/logstash/public/services/license/logstash_license_service.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export class LogstashLicenseService {
+  constructor(license, navigateToApp, toasts) {
+    this.license = license;
+    this.navigateToApp = navigateToApp;
+    this.toasts = toasts;
+  }
+
+  get enableLinks() {
+    return this.calculated.enableLinks;
+  }
+
+  get isAvailable() {
+    return this.calculated.isAvailable;
+  }
+
+  get isReadOnly() {
+    return this.calculated.isReadOnly;
+  }
+
+  get message() {
+    return this.calculated.message;
+  }
+
+  get isSecurityEnabled() {
+    return this.license.getFeature(`security`).isEnabled;
```

Comment:
NIT: ` -> '

Reply:
Can't use arrow functions on `get` properties (if that's what you meant).